src/scala/com/github/johnynek/bazel_deps/Decoders.scala

```diff
@@ -15,6 +15,12 @@ object Decoders {
       case "runtime_deps" => Right(Transitivity.RuntimeDeps)
       case other => Left(s"unrecogized transitivity: $other")
     }
+  implicit val resolverCacheDecoder: Decoder[ResolverCache] =
+    Decoder.decodeString.emap {
+      case "local" => Right(ResolverCache.Local)
+      case "bazel_output_base" => Right(ResolverCache.BazelOutputBase)
```

Comment:
My only concern with this encoding is that if someone wants to give a full path we don't have a way to disambiguate deterministically.

I guess we could do something like if it starts with "path:" we truncate that and use the rest as a literal path. The reason I'm thinking of this is we have several repos. It might actually be nice to use a shared cache between them (something I hadn't bothered to think of).

What do you think of this?